src/schema/rules/dataset_metadata.yaml

```diff
@@ -0,0 +1,53 @@
+dataset_description:
+    selectors:
+    - path == /dataset_description.json
+    fields:
+        Name: REQUIRED
+        BIDSVersion: REQUIRED
+        DatasetType: RECOMMENDED
+        License: RECOMMENDED
+        Authors: OPTIONAL
+        Acknowledgements: OPTIONAL
+        HowToAcknowledge: OPTIONAL
+        Funding: OPTIONAL
+        EthicsApprovals: OPTIONAL
+        ReferencesAndLinks: OPTIONAL
+        DatasetDOI: OPTIONAL
+        GeneratedBy: RECOMMENDED
+        GeneratedBy[].Name: REQUIRED
+        GeneratedBy[].Version: RECOMMENDED
+        GeneratedBy[].Description: OPTIONAL
+        GeneratedBy[].CodeURL: OPTIONAL
+        GeneratedBy[].Container: OPTIONAL
+        GeneratedBy[].Container.Type: RECOMMENDED
+        GeneratedBy[].Container.Tag: RECOMMENDED
+        GeneratedBy[].Container.URI: RECOMMENDED
+        SourceDatasets: RECOMMENDED
+
+derivative_description:
+    selectors:
+    - path == /dataset_description.json
+    - dataset.dataset_description.DatasetType == "derivative"
```

Comment:
In this case, wouldn't the `DatasetType` be available through the targeted file (i.e., the dataset_description)?

Reply:
Sure, but what would that look like?

Reply:
data.DatasetType I guess?

Reply:
`json.DatasetType`